docs/concepts/resolver.md

```diff
@@ -4,4 +4,38 @@ sidebar_position: 2
 
 # Resolvers
 
-TODO
\ No newline at end of file
+
+The `resolvers` option is used to configure resolvers, default being `I18nJsonLoader`.The resolvers takes in array of resolvers so you can add in any number of resolvers you like.
```

Comment:
Resolves and loaders are two different things. There are no default resolvers being set.

Reply:
Thats just me copy pasting. Will fix it